www_src/lib/api.js

```diff
@@ -1,22 +1,24 @@
-//var xhr = require('xhr');
-var xhr = require('./api-mock').getResponse;
+var xhr = require('xhr');
+//var xhr = require('./api-mock').getResponse;
 var defaults = require('lodash.defaults');
-var BASE_URL = '';
+var BASE_URL = 'https://webmaker-api.herokuapp.com';
+// var BASE_URL = 'http://localhost:2015';
 var mocks = require('./api-mock');
 
 module.exports = function (options, callback) {
   // Set default options
   defaults(options, {
-    method: 'get',
+    method: 'GET',
     useCache: false,
     json: {},
-    timeout: 60000    // 60 seconds
+    headers: {},
+    timeout: 60000 // 60 seconds
```

Comment:
I think for our expectations on 3G / 4G networks it is, but I'm worried about setting this two low for Edge networks in places like Bangladesh / China.